RobotsWantedLeague/Services/NotEmptyRobotsService.cs

```diff
@@ -7,28 +10,39 @@ public class NotEmptyRobotsService: IRobotsService
     private readonly IRobotsService underlyingRobotsService;
     public List<Robot> Robots { get => underlyingRobotsService.Robots; }
 
-    public NotEmptyRobotsService()
+    public NotEmptyRobotsService(RobotsService underlyingRobotsService)
```

Comment:
We changed a little bit how we build `NotEmptyRobotsService`.  Now we ask ASP to auto wire the `RobotsService` for us.  We need to do this because `RobotsService` depends on mongodb which can only be build via ASP auto wiring mechanism.